### DISCUSSION TITLE
Where to discuss Smart Connect?

### DISCUSSION THREAD

**Discussion by karkojk** [brianpetro/smart-connect#14] [Help & Support]
I couldn't find an already developed discussion around smart conect. Could there be a link to https://github.com/brianpetro/obsidian-smart-connections/discussions

**Comment by brianpetro** [comment]
Hi @karkojk 

The above-linked discussions are specifically for Smart Connections, the Obsidian plugin. 

There is a lot of overlap, and Smart Connect is relatively new, so either place would be fine for Smart Connect discussions.

However, in time, I will move the Smart Connect discussions to be focussed here 🌴
